File: l2k-server-game-model/src/main/java/org/l2kserver/game/model/actor/character/PvpState.java
package org.l2kserver.game.model.actor.character;

/**
 * Player's PvP state
 */
public enum PvpState {
    /** Player is not in PvP */
    NOT_IN_PVP,

    /** Player is in PvP */
    PVP,

    /** Player's PvP state is going to end (blinking violet name) */
    PVP_ENDING
}

File: l2k-server-game-model/src/main/kotlin/org/l2kserver/game/model/actor/character/PvpState.kt
package org.l2kserver.game.model.actor.character

/**
 * Player's PvP state
 *
 * @property NOT_IN_PVP Player is not in PvP
 * @property PVP Player is in PvP
 * @property PVP_ENDING Player's PvP state is going to end (blinking violet name)
 */
enum class PvpState {
    NOT_IN_PVP,
    PVP,
    PVP_ENDING
}
